django-stubs/utils/decorators.pyi

```diff
@@ -16,7 +15,7 @@ def method_decorator(decorator: Callable | Iterable[Callable], name: str = ...)
 def decorator_from_middleware_with_args(middleware_class: type) -> Callable: ...
 def decorator_from_middleware(middleware_class: type) -> Callable: ...
 def make_middleware_decorator(middleware_class: type[MiddlewareMixin]) -> Callable: ...
-
+@type_check_only
```

Comment:
Two things:
1. Looks like this should be internal: `_AsyncGetResponseCallable`
2. Can you please also fix https://github.com/typeddjango/django-stubs/issues/1269 while we are at it?

Reply:
Made it internal, moved it alongside a usage on MiddlewareMixin.